addOns/graphql/src/test/java/org/zaproxy/addon/graphql/VariantGraphQIUnitTest.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2020 The ZAP Development Team
```

Comment:
2023

Reply:
Hey! I resolved this only now. Sorry for missing it earlier.

Reply:
This seems fixed, but I don't see the CHANGELOG update having been pushed yet.

Reply:
I have pushed the updated CHANGELOG. Any corrections?